### PR TITLE
{data}[foss/2021b] scikit-learn v1.0.2 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by mustafabar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by mustafabar** [comment]
Test report by @mustafabar
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/2e91ff361e625a4cdae83dd3c938fb8c for a full test report.

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@mustafabar this can be added, but note that version `1.0.1` is already being used as a dependency in this toolchain version, so merging future easyconfigs that uses version `1.0.2` as a dependency in `2021b` will require an exception to the tests. Do you have a particular use case in mind?

otherwise, it may make more sense to wait for 2022a (?)

**Comment by mustafabar** [comment]
@migueldiascosta I have no particular use case except that the latest version was requested by one of our users so I thought I will share it here, but I guess it can wait to 2022a

File: easybuild/easyconfigs/s/scikit-learn/scikit-learn-1.0.2-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-learn/scikit-learn-1.0.2-foss-2021b.eb:1]: @mustafabar Please change this to a bundle of `scikit-learn` + the `sklearn` metapackage, like we did in #15613

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3131.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/4e13e674d2494916af21dd1eadebe923 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @mustafabar!

**Pull request merged**